### ISSUE TITLE
Auto-reload newly added feeds

### ISSUE THREAD

**Issue by Minoru** [newsboat/newsboat#1493]
New users sometimes get confused: they add a feed, immediately try to enter it, only to see the "Feed contains no items!" error. I think it'd be a better user experience if empty feeds were automatically reloaded on Newsboat startup, after `reload-urls` and `edit-urls` ops.

**Comment by bambirombi** [comment]
Yes, it would be great! I was to create a new issue because I was finding that same error. Is there any way to configure it that way?

**Comment by Minoru** [comment]
@bambirombi It's impossible to configure Newsboat in this way yet, because this hasn't been implemented. I'll merge a pull request that adds it :)
